futurecollars/src/main/java/com/futurecollars/lekcja6/zadanie7/Element.java

```diff
@@ -0,0 +1,19 @@
+package com.futurecollars.lekcja6.zadanie7;
+
+public class Element {
+    private double number;
+
+    public Element(double number) {
+        this.number = number;
+    }
+
+
+    public double changeNumberIntoRoot() {
```

Comment:
Proponuję nazwę dla tej metody "calculateRoot" to "change" jest troche mylące ;)